Documentation/devel/stage1-implementors-guide.md

```diff
@@ -80,6 +80,14 @@ Any stage1 that supports and expects machined registration to occur will likely
 * `--disable-paths` disables inaccessible and read-only paths (such as `/proc/sysrq-trigger`)
 * `--disable-seccomp` disables seccomp (overrides `retain-set` and `remove-set`)
 
+#### Arguments added in interface version 4
+
+* `--dns-conf-mode=resolv=(host|stage0|none|default),hosts=(host|stage0|default)`: Configures how the stage1 should set up
```

Comment:
If the two are flags can be used independently (and it looks to me they can), I'd prefer to keep stage1 interface easier to implement by splitting this into two switches.